packages/odyssey-react/src/components/TextInput/TextInput.tsx

```diff
@@ -107,13 +108,7 @@ interface CommonProps
   onFocus?: FocusEventHandler<HTMLInputElement>;
 }
 
-interface SearchProps extends Omit<CommonProps, "type" | "prefix" | "suffix"> {
-  type: "search";
-  prefix?: never;
-  suffix?: never;
-}
-
-export type TextInputProps = CommonProps | SearchProps;
+export type TextInputProps = CommonProps;
```

Comment:
we can remove CommonProps type entirely now and just export TextInputProps above

Reply:
👍🏻